static/app/views/starfish/views/spans/types.tsx

```diff
@@ -45,3 +45,23 @@ export const getTooltip = (
   }
   return '';
 };
+
+export const getThroughputTitle = (spanOp?: string) => {
+  if (spanOp?.startsWith('db')) {
+    return t('Queries');
+  }
+  if (spanOp) {
+    return t('Requests');
+  }
+  return '--';
```

Comment:
Is it possible that the span op is something like middleware here?

Reply:
Hey @DominikB2014 , yeah it's possible for span op to be middleware or any other of the generic span ops. This change is intentional since we want to prioritize for DB spans and HTTP spans first. ie db spans will be labeled `Queries`, while all other spans will be labeled `Requests` for now. We intend to revisit this in the future.